stdlib/ext/async-ext.mc

```diff
@@ -0,0 +1,27 @@
+
+
+
+
+
+type Promise
+
+external asyncSleep ! : Float -> Promise a
```

Comment:
It would be nice if naming reveals that the unit is seconds (and not ms, for example). What about `asyncSleepSec`?